app/models/shoutout.rb

```diff
@@ -30,6 +30,8 @@ class Shoutout < ApplicationRecord
 
   validates :rich_text, presence: true, uniqueness: { scope: %i[user_id time_period_id] }
 
+  scope :not_celebrate, -> { where(type: nil) }
+
   def to_text
     rich_text.gsub(%r{<span class="color-primary">|</span>}, '')
```

Comment:
Maybe it's better to use `strip_tags(rich_text)`

Reply:
@LyubovP this scope needs for separating CelebrateShoutouts from Shoutouts, I am suer that strip_tags(rich_text) can not help with it.